packages/@aws-cdk/aws-codebuild/test/integ.project-secondary-sources-artifacts.expected.json

```diff
@@ -209,6 +209,11 @@
             "SourceIdentifier": "AddSource1",
             "Type": "S3"
           }
+        ],
+        "SecondarySourceVersions": [
+          {
+            "SourceIdentifier": "AddSource1"
+          }
```

Comment:
Hmm. This is a problem.

I think you should filter rendering on whether `sourceVersion` is !== `undefined`. For example, this property should not be rendered in this case.

Reply:
I have updated so `SecondarySourceVersions` won't render if no versions are given.